src/Fractions/Fraction.Constructors.cs

```diff
@@ -83,19 +76,15 @@ public Fraction(BigInteger numerator, BigInteger denominator)
     /// <param name="numerator">integer value that will be used for the numerator. The denominator will be 1.</param>
     [CLSCompliant(false)]
     public Fraction(ulong numerator) {
-        Numerator = new BigInteger(numerator);
-        _denominator = BigInteger.One;
-        State = FractionState.IsNormalized;
+        _numerator = new BigInteger(numerator);
     }
 
     /// <summary>
     /// Creates a normalized fraction using a big integer.
     /// </summary>
     /// <param name="numerator">big integer value that will be used for the numerator. The denominator will be 1.</param>
     public Fraction(BigInteger numerator) {

```

Comment:
Is there a reason why the denominator is not explicitly set to `BigInteger.One`?